app/src/main/java/ch/epfl/swissteam/services/DBUtility.java

```diff
@@ -173,11 +173,12 @@ public void onCancelled(@NonNull DatabaseError databaseError) {
 
     /**
      * Retrieves the POSTS_DISPLAY_NUMBER freshest post of the database in geographical range of the user.
-     *
-     * @param callBack     the function called on the callBack
+     *  @param callBack     the function called on the callBack
      * @param userLocation the location of the user
+     * @param currentUserId
+     * @param helper
      */
-    public void getPostsFeed(final MyCallBack<ArrayList<Post>> callBack, Location userLocation) {
+    public void getPostsFeed(final MyCallBack<ArrayList<Post>> callBack, Location userLocation, String currentUserId, SettingsDbHelper helper) {
```

Comment:
You can retrieve the userID from GoogleIDSingleton without needing to pass it as argument, could be simpler :)